ingestion/src/metadata/orm_profiler/profiler/handle_partition.py

```diff
@@ -132,7 +132,8 @@ def build_partition_predicate(partition_details: dict, col):
             f"{partition_details['partition_field']} in "
             f"({format_partition_values(partition_details['partition_values'], col_type)})"
         )
-
+    if partition_details["partition_field"] == "_PARTITIONDATE":
+        col_type = "DATE"
```

Comment:
what if it's `_PARTITIONDATETIME`?

Reply:
`_PARTITIONTIME` sends a column name as partition_field

`_PARTITIONDATE` sends the column name as None, so need to add col_type as `Date`

Reply:
🤔 are you sure? As you can see below we are passing the column by getting it from the list of columns on the SQA Declarative Table. Since `_PARTITIONDATE`, `_PARTITIONTIME` I believe they both are not in the columns of the declarative table.
https://github.com/open-metadata/OpenMetadata/blob/c1e6e947dc08428d1ece6777e7f58ab1be75ac45/ingestion/src/metadata/orm_profiler/profiler/handle_partition.py#L171

Reply:
If it's `_PARTITIONTIME` for partition interval set as `DATE`
we get: 
```
col = {Column} asd.e
col_type = {DATE} DATE
partition_details = {dict: 4} {'partition_field': 'e', 'partition_start': datetime.datetime(2022, 8, 23, 0, 0), 'partition_end': datetime.datetime(2022, 9, 23, 0, 0), 'partition_values': None}
```


but if it's `_PARTITIONDATE`
we get:
```
col = {NoneType} None
col_type = {NoneType} None
partition_details = {dict: 4} {'partition_field': '_PARTITIONDATE', 'partition_start': datetime.datetime(2022, 8, 23, 0, 0), 'partition_end': datetime.datetime(2022, 9, 23, 0, 0), 'partition_values': None}
```

Reply:
so if it's `_PARTITIONDATE `, I set the col_type as `Date` so that the predicate it creates is just for `Date` and not `DateTime`